DOCUMENTATION.md

```diff
@@ -10,83 +29,86 @@ If you want to read more into Flatseal's permissions, you can look at the [Permi
-[Inter-process communications](https://en.wikipedia.org/wiki/Inter-process_communication) | Toggle | Share or unshare IPC namespace with the host. | `--share=ipc` and `--unshare=ipc`
+Network | Toggle | Allow the application to have access to the network. <br /> <br /> For example, if it's disabled for Firefox, you will no longer be able to browse the internet. | `--share=network` and `--unshare=network`
+[Inter-process communications](https://en.wikipedia.org/wiki/Inter-process_communication) | Toggle | Share IPC namespace with the host. <br /> <br /> This is required by X11 due to it depending on IPC. Even if it's enabled, this will not make it any more dangerous. | `--share=ipc` and `--unshare=ipc`
 
 #### Socket
 
 List of well-known sockets available in the sandbox.
 
 Name | Type | Description | `flatpak override` equivalent
 --- | --- | --- | ---
-X11 windowing system | Toggle | Allow or prohibit access to the application to show windows using X11. | `--socket=x11` and `--nosocket=x11`
-Wayland windowing system | Toggle | Allow or prohibit access to the application to show windows using Wayland. | `--socket=wayland` and `--nosocket=wayland`
-Fallback to X11 windowing system | Toggle | Allow or prohibit access to the application to show windows using X11 if Wayland is not available. **This overrides the X11 windowing system option when enabled.** | `--socket=fallback-x11` and `--nosocket=fallback-x11`
-PulseAudio sound server | Toggle | Allow or prohibit the application to play sounds that use PulseAudio. | `--socket=pulseaudio` and `--nosocket=pulseaudio`
-D-Bus session bus | Toggle | Allow or prohibit access to the application to the entire session bus. | `--socket=session-dbus` and `--nosocket=session-dbus`
-D-Bus system bus | Toggle | Allow or prohibit access to the application to the entire system bus. | `--socket=system-dbus` and `--nosocket=system-dbus`
-Secure Shell agent | Toggle | Allow or prohibit access to the application to SSH authentications. | `--socket=ssh-auth` and `--nosocket=ssh-auth`
-Smart cards | Toggle | Allow or prohibit access to the application to smart cards. | `--socket=pcsc` and `--nosocket=pcsc`
-Printing system | Toggle | Allow or prohibit access to the application to printing systems. | `--socket=cups` and `--nosocket=cups`
+X11 windowing system | Toggle | Allow the application to open in an X11 window. <br /> <br /> Most applications typically use X11 as it is a more mature display protocol. | `--socket=x11` and `--nosocket=x11`
+Wayland windowing system | Toggle | Allow the application to open in a Wayland window. <br /> <br /> Many applications do not use Wayland as it is a newer display protocol unlike X11. Most of the ones written in GTK are natively compatible with Wayland. Some require the user to use an environment variable to enable Wayland; `MOZ_ENABLE_WAYLAND=1` in Firefox's case. | `--socket=wayland` and `--nosocket=wayland`
```

Comment:
I don't think most apps using x11 because is more mature, it's mostly for historic reasons.

We should probably shorten that extra comment a bit, e.g.

re. X11, ... Most applications use X11 for historic reasons, but is considered less secure.
re. Wayland, ... Is a newer protocol, considered more secure, but some applications require extra steps to use it (See environment variables example).

Reply:
Is it good?